### ISSUE TITLE
[Prooftool] "Hide structural rules" makes no sense

### ISSUE THREAD

**Issue by loewenheim** [gapt/gapt#478]
The current semantics is that the premises of structural rules will be displayed and the conclusions hidden, when it should be the other way around. Consider:
```
        P(s), ∀x. P(x) :- Q
    ----------------------------∀:l
     ∀x. P(x), ∀x. P(x) :- Q
    -----------------------------c:l
            ∀x. P(x) :- Q
```

What you would expect to see after checking "Hide structural rules" is
```
      P(s), ∀x. P(x) :- Q
    --------------------------∀:l
           ∀x. P(x) :- Q
```
But what you get is
```
        P(s), ∀x. P(x) :- Q
    ----------------------------∀:l
     ∀x. P(x), ∀x. P(x) :- Q
```
That doesn't even have the same end sequent as the original proof! With weakening it's similar.

**Comment by loewenheim** [comment]
Update: I think what I did in e3fba7ff07b5ab6eb2c6952ba0ea929a94df197b is correct, but it displays really weirdly on my system. Please tell me how it looks to you.

**Comment by loewenheim** [comment]
Fixed in https://github.com/gapt/gapt/commit/cbc992a9fc3f9d0361714b5c5674495a68f255fe.

**Comment by gebner** [comment]
This is not yet in master, right?

**Comment by loewenheim** [comment]
No. Should I not have closed it?

**Comment by gebner** [comment]
It's fine.  I just usually keep bugs open until they're merged, so that I don't forget to merge them--that's why I was a bit puzzled.

**Comment by loewenheim** [comment]
I see. Is that why github didn't automatically close the issue despite the commit message saying "Fixes #478"?

**Comment by gebner** [comment]
Because the commit is not merged yet.  The issue is only closed when the commit with the "Fixes #xyz" message is merged into master.
